### ISSUE TITLE
Difference in compiled JS in between deck and VS - Possible symbol resolving failed

### ISSUE THREAD

**Issue by beercsaba** [bridgedotnet/Bridge#2213]
[forum link](http://forums.bridge.net/forum/community/help/3240-difference-in-compiled-js-in-between-deck-and-vs)

### Expected

```js
var dataTable = $(idSelector).DataTable( { "language": { "url": "//cdn.datatables.net/plug-ins/1.10.13/i18n/Hungarian.json" } } ) ;
```

### Actual
```
TypeError: Cannot read property 'script' of undefined
 /RunHandler.ashx?h=1599467679:23 Function.getTable()
```
```js
var dataTable = Bridge.bridge.script.write("$(idSelector).DataTable( { \"language\": { \"url\": \"//cdn.datatables.net/plug-ins/1.10.13/i18n/Hungarian.json\" } } ) ");
```

### Steps To Reproduce

http://deck.net/dd360cb16b6e3390243ae4d8f0909653

```cs
using Bridge;

namespace Issue1
{
    public class UIUtils
    {
        public static void Main()
        {
            var t = UIUtils.DataTables.GetTable();
            
            // Sould be 1
            System.Console.WriteLine(t.i);
        }
        
        public class DataTables
        {
            public static DataTable GetTable()
            {
                return Script.Write<DataTable>("{i:1}");
            }
        }

        public class DataTable
        {
            public int i;
        }
    }
}
```

**Comment by vladsch** [comment]
Thanks for the report
Here is simpified sample which reproduces the issue
```
using Bridge;

namespace Issue1
{
    public class UIUtils
    {
        public class DataTables
        {
            public static DataTable GetTable()
            {
                return Script.Write<DataTable>("{}");
            }
        }

        public class DataTable
        {
        }
    }
}
```

**Comment by beercsaba** [comment]
it's an almost nude sample :)

**Comment by beercsaba** [comment]
sorry, i did closed it mistakenly...
i just wanted to write that it's almost nude sample :)

**Comment by geoffreymcgill** [comment]
Hi @beercsaba – Thanks for reporting this issue. We are reviewing this issue. With some luck, a fix can be included in the upcoming Bridge 15.7 release.

**Comment by beercsaba** [comment]
Thanks, guys!

**Comment by geoffreymcgill** [comment]
Fix has been added to *master* branch and will be included in Bridge 15.7.0.
